viewing_party/party.py

```diff
@@ -1,23 +1,276 @@
 # ------------- WAVE 1 --------------------
+from collections import Counter
+from enum import unique
 
+# inputs a movie into the program
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+        return movie
+    else:
+        return None
+
+# takes a movie dict and adds it to user's watched
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data
+
+# takes a movie dict and adds it to user's watchlist
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data
+
+# finds the title of a movie in watchlist, then removes and transfers it to watched
+def watch_movie(user_data, title):
+    counter = 0
+    for movie in user_data["watchlist"]:
+        if title in movie["title"]:
+            user_data["watched"].append(movie)
+            del(user_data["watchlist"][counter])
+        # if title in user_data["watchlist"]:
+            # how to remove the title from the watchlist
+            # movie_seen = user_data["watchlist"].pop(title)
+            # movie_seen = user_data.pop("watchlist")
+            # user_data["watched"][movie] = [movie_seen]
+            return user_data

```

Comment:
This `return` is extraneous since on line 41, you return `user_data`. If you're wanting to return early from the loop itself, you could you use `break` instead.